kv/memberlist/memberlist_client.go

```diff
@@ -135,6 +135,10 @@ type KVConfig struct {
 	DeadNodeReclaimTime time.Duration `yaml:"dead_node_reclaim_time"`
 	EnableCompression   bool          `yaml:"compression_enabled"`
 
+	// ip:port to advertise other cluster members. Used for NAT traversal
```

Comment:
Could you rebase please? These changes should go away from this PR.

Reply:
@pracucci will do.